src/rabbit_memory_monitor.erl

```diff
@@ -86,7 +86,9 @@ report_ram_duration(Pid, QueueDuration) ->
 stop() ->
     gen_server2:cast(?SERVER, stop).
 
-conserve_resources(Pid, disk, Conserve) ->
+%% Paging should be enabled/disabled only in response to disk resource alarms
```

Comment:
as discussed on slack with @dumbbell what would happen when node A that doesn't have an alarm, receives a publish that has to be routed or replicated to node B (who has a disk alarm for instance). What to do here?